Doc/whatsnew/3.7.rst

```diff
@@ -336,6 +336,10 @@ Build and C API Changes
   is now of type ``const char *`` rather of ``char *``. (Contributed by Serhiy
   Storchaka in :issue:`28769`.)
 
+* The result of :c:func:`PyMapping_Keys`, :c:func:`PyMapping_Values` and
+  :c:func:`PyMapping_Items` is now always a list, rather than a list or a
+  tuple. (Contributed by Serhiy Storchaka and Oren Milman in :issue:`28280`.)
```

Comment:
No need to mention my name. I'm just a reviewer.

Reply:
You are the author of the issue in bpo, but as you wish.

Reply:
The one who opened an issue is not the one who fixed it.